examples/gpt/single/run_eval.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import math
+import time
+import sys
+import random
+import numpy as np
+
+import paddle
+from paddle.io import DataLoader
+
+sys.path.append("../../../")
+from fleetx.models.gpt_model.modeling import GPTModel, GPTForPretraining
+from fleetx.data.tokenizers import GPTTokenizer
+from fleetx.data.sampler import Stack, Tuple
+from fleetx.datasets.lm_test import LM_Eval_Dataset, Lambada_Eval_Dataset, wikitext_detokenizer
+from fleetx.utils import logger
+from examples.gpt.tools import parse_yaml
+
+
+def get_tokens(tokenizer, text, strict=True):
```

Comment:
Add document about:
1. what this function do
2. type of inputs
3. type of return value and the meaning of them.